content/services/opence.rst

```diff
@@ -0,0 +1,186 @@
+Open-CE
+=======
+:slug: opence
+:author: Ken Lett
+:menu: POWERLinux/OpenPOWER Development Hosting, Open-CE, 3
+
+The Open Source Lab (OSUOSL) and Center for Genome Research and Biocomputing (CGRB) partner with IBM and OpenPOWER in order to provide a download resources around Open-CE. Open-CE is a community driven software distribution for machine learning that runs on standard Linux platforms with NVIDIA GPU technologies.
+- `Current release`_
+
+- `Previous releases`_
+
+.. _Current release:
+.. _Release 1.0.0:
+
+Open-CE Release 1.0.0
+---------------------
+
+*Release date: 11/10/2020*
+
+**What's new**
+
+Open-CE 1.0 is the `current release`_ of Open-CE and includes the following features:
+
+- conda packages are now available on ppc64le.
+- conda packages are now available on x86.
+- TensorFlow 2.3.1
+- PyTorch 1.6.0
+- Open-CE is distributed as prebuilt containers, or on demand through the Conda provisioning process.
+
+  - All of the Conda packages are available in a `Open-CE Conda channel`_
+  - Current Conda packages are available in a `Current Open-CE Conda channel`_
+  - There is no install package to download, instead connect to the Conda channel and install your packages from there
+  - Package dependencies are automatically resolved
+  - Delivery of packages is open and continuous
+  - Enable Python versions 3.6, 3.7, 3.8
+  - You can run more than one framework at the same time in the same environment. For example, you can run TensorFlow and PyTorch at the same time.
+
+.. _Open-CE Conda channel: https://ftp.osuosl.org/pub/open-ce/
+.. _Current Open-CE Conda channel: https://ftp.osuosl.org/pub/open-ce/current
+
+**Learn more**
+
+Get information about planning, configuring, and managing Open-CE 1.0 Below:
+
+- `Planning`_
+- `System setup`_
+- `Setting up the software repository`_
+- `Creating conda environments`_ (recommended)
+- `Installing the MLDL frameworks`_
+- `Uninstalling the MLDL frameworks`_
+
+.. _planning:
+
+Planning
+--------
+
+We recommend users use one of the listed operating systems listed above. This is a standard conda repository and can be added to any conda install. Conda must be configured to give priority to installing packages from this channel.
```

Comment:
Where is the list of operating systems you mention here?

Reply:
Good question! I just copied Chris's content in and formatted, so maybe I lost a section or he forgot to add that, i will check

Reply:
updated to refer to the list below, not above